src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -18,6 +18,9 @@ const chatMessageStyles: IComponentPartStylesInput<IChatMessageProps, IChatMessa
           marginRight: 'auto',
         }),
     maxWidth: v.messageWidth,
+    width: 'max-content',
```

Comment:
could you, please, suggest what would be the styling problem if this CSS property won't be applied? The reason I am asking is because this one is marked as experimental: https://developer.mozilla.org/en-US/docs/Web/CSS/width

Reply:
Without that prop this is how the chat looks:
![image](https://user-images.githubusercontent.com/4512430/46288938-ca4b7600-c587-11e8-9661-ee6f514839f2.png)
I will try to implement this behavior using some other styling then...